trussworks/react-uswds
# docs(Storybook): 404 page template added

# Summary
This pr adds a 404 page not found example to the ReactUSWDS Storybook like the one seen in USWDS' documentation. As many of the elements of the template already existed in the documentation template and the authentication templates I borrowed chunks of code from those examples and updated with the appropriate changes. 

## Related Issues or PRs

https://github.com/trussworks/react-uswds/issues/1619

## How To Test

* Run "yarn storybook" to start Storybook locally. 
* Can verify addition of template by navigating to "page templates > not found page". 
* Can then compare template addition with the USWDS 404 example found [here](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.app.cloud.gov/preview/uswds/uswds/develop/components/preview/layout--404.html).


### Screenshots (optional)
<img width="1143" alt="Screen Shot 2022-02-25 at 5 18 05 PM" src="https://user-images.githubusercontent.com/20867088/155810757-e7474cf5-93d1-448f-b861-ba26379fe115.png">
<img width="1145" alt="Screen Shot 2022-02-25 at 5 18 15 PM" src="https://user-images.githubusercontent.com/20867088/155810760-36f8b329-2e53-4242-a15f-0718c9824d54.png">
